src/common/constants.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export const COMMON_CONSTANTS = {
```

Comment:
Should it be the responsibility of the initializers to provide the database and store names? This seems like an internal detail, especially if we're always passing the same values. Would it make sense instead to hide them inside a function that wraps the constructor?